stable/artifactory-ha/templates/artifactory-gcp-credentials-secret.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if not .Values.artifactory.persistence.googleStorage.gcpServiceAccount.customSecretName }}
```

Comment:
Please ass an annotation with the checksum of this file, similar to what we did with the system.yaml